### PR TITLE
add front end test description

### PR DESCRIPTION
**Pull request by rafaelDynamox**
Including front end test description and resources.

### CODE REVIEW

File: front-end-challenge.md [hunk 1]
```diff
@@ -2,4 +2,79 @@
...
 
-<!--TODO-->
\ No newline at end of file
+Dynamox front-end development team presents you with the following challenge:
+
+**Using React and TypeScript, develop a robust and intuitive dashboard application that allows assessing the data acquired by our sensors.**
+
+---
+
+Keep in mind the challenge aims to reproduce an environament where you could demonstrate your skills.
```
Comment (andrespilleredyn) [inline front-end-challenge.md:11]: environment

File: front-end-challenge.md [hunk 2]
```diff
@@ -2,4 +2,79 @@
...
+
+In order to help guiding your development process we will provide some requirements. It is not mandatory to fulfill all requirements to submit your implementation. The more requirements you implement, the more resources we will have to assess your skills and knowledge. 
+
+Use your best judgement to prioritize tasks to meet the time you have available. Feel free to make any assumptions you consider necessary to complete the task.
+
+## Functional Requirements and User Stories
+
+In maintenance industry, vibration analysis plays a main role: It uses physical quantities like acceleration and velocity to find evidences that help predict occurence of faults or degradations in machines. 
+
+The page that displays this information in Dynapredict, our asset condition monitoring platform, is designed pretty much like [this](https://www.figma.com/file/QxUZkTUIzQA7cvyiMvVyxK/Front-end---Teste?type=design&node-id=1001%3A3&mode=design&t=JLnbGmQJcSlnYYE2-1).
```
Comment (andrespilleredyn) [inline front-end-challenge.md:21]: DynaPredict

File: response.json [hunk 3]
```diff
@@ -0,0 +1,5104 @@
+[{
+    "name": "acceleration/x/continuous/rms",
```
Comment (andrespilleredyn) [inline response.json:2]: accelerationRms/x

File: front-end-challenge.md [hunk 4]
```diff
@@ -2,4 +2,79 @@
...
+
+
+2 - Technical requirements
+1. [ ] Use TypeScript.
+1. [ ] Use React.
+1. [ ] Use Redux for managing global states. [Check out](https://redux-toolkit.js.org/introduction/getting-started) to get started.
+1. [ ] Use Redux Saga for asynchronous side effects. [Check out](https://redux-saga.js.org/docs/introduction/GettingStarted) to get started.
+1. [ ] Use Vite for building the app. [Check out](https://vitejs.dev/guide/) to get started.
+1. [ ] Use Material UI 5 for styling the application. [Check out](https://mui.com/material-ui/getting-started/) to get started.
+1. [ ] Use Highcharts to display the plots. [Check out](https://www.highcharts.com/docs/index) to get started.
```
Comment (andrespilleredyn) [inline front-end-challenge.md:42]: plotly, D3, or any similar library.

**Review by calil-amaral-dynamox — APPROVED** [review]

**Pull request merged**